test/feature/reverse_routing.feature

```diff
@@ -38,18 +39,28 @@ Feature: reverse routing
     Then I have a routing helper "<HELPER>" that returns "<VALUE>"
 
     Examples:
-      | HELPER                         | VALUE           |
-      | res.locals.paths.users()       | /users          |
-      | res.locals.paths.user(123)     | /users/123      |
-      | res.locals.paths.newUser()     | /users/new      |
-      | res.locals.paths.editUser(123) | /users/123/edit |
+      | HELPER                                    | VALUE           |
+      | res.locals.routes.users()                 | /users          |
+      | res.locals.routes.user(123)               | /users/123      |
+      | res.locals.routes.newUser()               | /users/new      |
+      | res.locals.routes.editUser(123)           | /users/123/edit |
+      | res.locals.routes.updateUser(123)         | /users/123      |
+      | res.locals.routes.createUser()            | /users          |
+      | res.locals.routes.destroyUser(123)        | /users/123      |
+      | res.locals.routes.users().method          | GET             |
+      | res.locals.routes.user(123).method        | GET             |
+      | res.locals.routes.newUser().method        | GET             |
+      | res.locals.routes.editUser(123).method    | GET             |
+      | res.locals.routes.updateUser(123).method  | PUT             |
+      | res.locals.routes.createUser().method     | POST            |
+      | res.locals.routes.destroyUser(123).method | DELETE          |
 
 
-  Scenario: paths object can be passed in to options
+  Scenario: routes object can be passed in to options
     Given the routing definition
       """
       GET '/some/route', to: 'test#index', as: 'foobar'
       """
-    And an exprestive app with the option "paths" set to `global.custom_paths_object = {}`
-    Then I have a routing helper "custom_paths_object.foobar()" that returns "/some/route"
-    And the routing helper "res.locals.paths" is undefined
+    And an exprestive app with the option "routes" set to `global.custom_routes_object = {}`
```

Comment:
Unfortunately at this time this step definition is what boots up the server so it can't go above the setup.  We talked a bit about this last Thursday. I can refactor the server to boot up on the first `When` or `Then` clause in a future PR